pkg/exec/run.go

```diff
@@ -137,6 +137,7 @@ func PrepareCommand(
 	version v0manifest.Version,
 	binPath, tag, instanceDir string, wd string,
 	args []string,
+	sysProcAttr *syscall.SysProcAttr,
```

Comment:
The function `PrepareCommand` has 11 parameters already, it's better to use a struct to define. How do you think?

Reply:
https://github.com/pingcap/tiup/pull/733/commits/3ddcf04d4ad043213bc46ad80855398018a92701